drivers/clock_control/clock_stm32_ll_u5.c

```diff
@@ -103,10 +99,8 @@ static uint32_t get_pllout_frequency(uint32_t pllsrc_freq,
 static uint32_t get_sysclk_frequency(void)
 {
 #if defined(STM32_SYSCLK_SRC_PLL)
-	return get_pllout_frequency(get_pllsrc_frequency(PLL1_ID),
-					STM32_PLL_M_DIVISOR,
-					STM32_PLL_N_MULTIPLIER,
-					STM32_PLL_R_DIVISOR);
+	return get_pllout_frequency(get_pllsrc_frequency(PLL1_ID), STM32_PLL_M_DIVISOR,
+				    STM32_PLL_N_MULTIPLIER, STM32_PLL_R_DIVISOR);
```

Comment:
This change makes code harder to read and maintain.

Reply:
abandoned